src/assets/i18n/ja-JP.json

```diff
@@ -295,8 +295,8 @@
       "enable": "天気を有効にする",
       "enableForecast": "予測を有効にする",
       "enableToday": "今日の天気を有効にする",
-      "feelsLike": "温度のように感じる",
-      "feelsLikeTitle": "のように感じている",
+      "feelsLike": "体感温度",
```

Comment:
Seems like the original is more accurate. So this line is for the weather widget, and it is to enable the option of a temperature that is based on what it "feels like" rather than the actual recorded temperature. Again, if yours is actually more accurate just let me know :)

Reply:
The word "体感温度" means as "Feeling temperature", and it is normally used for weather report.
Would you please check the website like "https://weather.com/" and change the language to Japanese,
I think you will find the word at the same place as "feels like".

![weather](https://user-images.githubusercontent.com/88008648/127084341-0d5380b5-b744-4c76-a92c-ca65e9c7cc20.png)

Reply:
Thanks for the extra info, sounds like it’s good then